Server/BackEndMonitoringServer/include/CLogicalDiskStatus.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "EMemoryCountType.h"
```

Comment:
The same as above

Reply:
Fixed, [commit](https://github.com/ITA-Dnipro/BackEndMonitoring/commit/28775d4e4e211a6985cecfb05174a510800a77d4#diff-0544ae2661f9747e70ea10c500758935c3f7d93fc11a7c69c55cadb18011b360)